packages/app/src/HDXLineChart.tsx

```diff
@@ -208,6 +210,14 @@ const MemoChart = memo(function MemoChart({
         {isClickActive != null ? (
           <ReferenceLine x={isClickActive.activeLabel} stroke="#ccc" />
         ) : null}
+        {logReferenceTimestamp != null ? (
+          <ReferenceLine
+            x={logReferenceTimestamp}
+            stroke="#ff5d5b"
+            strokeDasharray="3 3"
+            label="Log"
```

Comment:
```suggestion
            label="Event"
```
we should call it an event since it could be a span or log